migration/planning-migration-3-to-4.adoc

```diff
@@ -0,0 +1,158 @@
+
+// The introduction of OpenShift 4 represents a dramatic change for the way in which OpenShift clusters are deployed are managed by infrastructure and operational teams.  OCP4 includes new technologies and functionality which are required and core to the operation of the cluster, at the infrastructure layer these are most notably Operators, MachineSets and RHCoS (with that, immutable infrastructure).  The result of these technologies working in cohesion is an OpenShift cluster which is effectively self-aware and able to be managed from within itself.
+//
+// This paradigm shift is in stark contrast to the deployment and operation of an OpenShift v3 cluster today, where RHEL hosts are individually deployed and managed, with OpenShift v3 installed atop of these hosts in order to form a cluster.  Customer operations teams will traditionally still manage those RHEL hosts which are used for OpenShift v3 as a subset of their wider RHEL estate, managing nodes with common security baselines and/or standard software agents as well as applying other working practices to deploy, subscribe and maintain those hosts with existing patch windows.
+
+For detailed information on new features and other notable technical changes, review the xref:../release_notes/ocp-4-2-release-notes.adoc#ocp-4-2-release-notes[OpenShift Container Platform 4.2 release notes].
+
+[id="migration-differences-archictecture"]
+=== Architecture differences
+
+[discrete]
+==== Immutable infrastructure
+
+// TODO: Add in 3.11 details: RHEL / RHEL atomic?
+
+{product-title} 4 uses Red Hat Enterprise Linux CoreOS (RHCOS), which is designed to run containerized applications, and provides efficient installation, Operator-based management, and simplified upgrades. RHCOS is an immutable container host, rather than a customizable operating system like RHEL. RHCOS enables {product-title} 4 to manage and automate the deployment of the underlying container host. RHCOS is a part of {product-title}, which means that everything runs inside a container and is deployed using {product-title}.
+
+In {product-title} 4, control plane nodes must run RHCOS, ensuring that full-stack automation is maintained for the control plane. This makes rolling out updates and upgrades a much easier process than in {product-title} 3.
+
+For more information, see xref:../architecture/architecture-rhcos.adoc#architecture-rhcos[Red Hat Enterprise Linux CoreOS].
+
+[discrete]
+==== Operators
+
+Operators are a method of packaging, deploying, and managing a Kubernetes application. Operators ease the operational complexity of running another piece of software. They watch over your environment and use the current state to make decisions in real time. Advanced Operators are designed to upgrade and react to failures automatically.
+
+For more information, see xref:../operators/olm-what-operators-are.adoc#olm-what-operators-are[Understanding Operators].
+
+[id="migration-differences-install"]
+=== Installation and update differences
```

Comment:
@sdodson Can you please review these installation and upgrade sections and let me know whether it's accurate, or if anything is missing? Thanks!

Preview: https://migration-guide--ocpdocs.netlify.com/openshift-enterprise/latest/migration/planning-migration-3-to-4.html#migration-differences-install

Reply:
looks fine to me.